src/content/docs/tutorial-dd-migration/migrating-dashboards.mdx

```diff
@@ -22,7 +22,7 @@ import journeyAlertsAndDash from 'images/dashboards_screenshot-crop_journey-dash
 
 Setting up monitors for your data is the foundation of observability, but it's only half of the equation. You need to be able to visualize your data, group streams together, and be notified when your system runs into issues that negatively affect your users. You need to be able to do all this from a centralized location, and be able to adapt to the changing data needs of your organization by curating new visualizations on-the-fly. 
 
-Where New Relic entities can give you scores of data to observe, our dashboards and <InlinePopover type="alerts" /> come in to help you make the most of it. You can use our pre-configured options to setup solutions out of the box, or use [NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/), New Relic's Query Language, to create custom dashboard components and alert policies that are as fine-tuned to your requirements as you need want them to be. And like all our other features, you can try out dashboards and alerts using your own data [for free](https://newrelic.com/signup).
+Where New Relic entities can give you scores of data to observe, our dashboards and <InlinePopover type="alerts" /> come in to help you make the most of it. You can use our pre-configured options to set up dashboard solutions out of the box, or use [NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/), New Relic's Query Language, to create custom dashboard components and alert policies that are as fine-tuned to your requirements as you need want them to be. And like all our other features, you can try out dashboards and alerts using your own data [for free](https://newrelic.com/signup).
```

Comment:
```md
<InlinePoperover type="alerts" />
```
unsure if this needs to be updated or not. is the intent of this to add alerts, or just dashboards, or both via quickstarts?

Reply:
That popover code just points people to the alerts UI so they don't have to hunt it down themselves.